source/features/rgh-linkify-yolo-issues.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import {wrap} from '../helpers/dom-utils';
+import features from '../feature-manager';
+import {getRghIssueUrl} from '../helpers/rgh-issue-link';
+import {isRefinedGitHubYoloRepo} from '../github-helpers';
+
+function init(): void {
+	for (const issueCell of select.all('.js-csv-data td:nth-child(3)')) {
```

Comment:
This selector doesn't select anything

Reply:
![image](https://user-images.githubusercontent.com/118078892/216774291-ed75858d-6314-4bf9-bcd4-637896d218d4.png)

  i guess `awaitDomReady: false`  might be the issue the selector is working fine. I see the issues are linkified now.

Reply:
It's strange because that selector doesn't work for me, I tried it in the console

Reply:
okay so can i simply use `td:nth-child(3)`. or anything that you could suggest to make more specific.

Reply:
The feature itself will work on this repo and on this file only. Currently there is only one table so I guess using `td:nth-child(3)` should not be a issue. But in future if you are adding more tables than this one is showing csv data hence  the table has classes  like `js-csv-data csv-data` one of which I have used.  Other than this all tables will have same classes so not sure how to pick a specific class for this one. Let me know if I am getting it wrong or if you could suggest any better selector 🙂 .

Reply:
That's not specific enough, like before 

I'll try your class later again

Reply:
okay thanks 🙂

Reply:
<img width="643" alt="Screenshot 1" src="https://user-images.githubusercontent.com/1402241/216777433-f07833c9-ae2b-4d6b-966f-66a98989263d.png">

I see `.react-csv-row` though. Do you have this?

<img width="573" alt="Screenshot 3" src="https://user-images.githubusercontent.com/1402241/216777460-f7a9fbc8-abbe-44ef-ba17-d4d7fa1056f5.png">

Reply:
I think you're still seeing an older version of the UI. Maybe you can use this selector to target both:

```suggestion
	// .js-csv-data is the old selector
	for (const issueCell of select.all(':is(.js-csv-data, .react-csv-row) td:nth-child(3)')) {
```

Reply:
yes,  maybe i am seeing an older version of the UI. But thanks for making the changes.

![image](https://user-images.githubusercontent.com/118078892/216777683-1bc72919-9e5f-4b1c-9349-2d1187f45895.png)